include/aspect/material_model/rheology/peierls_creep.h

```diff
@@ -71,11 +115,43 @@ namespace aspect
                              const double temperature,
                              const unsigned int composition) const;
 
+          /**
+           * Compute the strain rate and first stress derivative
+           * as a function of stress based on the approximate Peierls creep law.
+           */
+          std::pair<double, double>
+          compute_approximate_strain_rate_and_derivative (const double stress,
+                                                          const double pressure,
+                                                          const double temperature,
+                                                          const PeierlsCreepParameters creep_parameters) const;
+
+          /**
+           * Compute the strain rate and first stress derivative
+           * as a function of stress based on the exact Peierls creep law.
+           */
+          std::pair<double, double>
+          compute_exact_strain_rate_and_derivative (const double stress,
+                                                    const double pressure,
+                                                    const double temperature,
+                                                    const PeierlsCreepParameters creep_parameters) const;
+
+          /**
+           * Compute the strain rate and first stress derivative
+           * as a function of stress based on the selected Peierls creep law.
```

Comment:
At first reading I was a bit confused about the difference between this function and the two above, but this became immediately clear after reading on. Perhaps worth adding a note that this function utilizes the two functions above?  If this seems excessive feel free to disregard.